src/components/switch-rect/switch-rect.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import SwitchBase from '../../base-components/switch-base'
+
+import './switch-rect.css'
+
+
+const SwitchRect = ({ id, checked, onChange, disabled, tabIndex, color }) => {
```

Comment:
the same concept as `switch-base/index.js`, this file should probably be named as `switch-rect/index.js`

Reply:
ooops true forgot!